README.md

```diff
@@ -4,22 +4,20 @@
 
 The Clean Dependency Project is Fannie Mae’s first open source project with a clear vision to provide clean OSS libraries to the products that we care about.  
 
+We started the Cleand Dependency Project to solve the problem of open source dependencies reporting a high and/or critical vulnerabilities, with a CVSS score of 9+ and no pubished fixes that patches these vulneralbilties. As our first step, we want to change these libraries to reduce the vulnerability surface area and second publish these patches publically so that others can benefit from the community work as well as contribute patches of their own. 
+
 There is an immediate concern with the consumption of OSS libraries that we rely on with no clear upgrade path.  The main goal is to clean these dependencies and vulnerabilities with patch management with the most recent versions to push back upstream into our projects as well as the external community.  
 
 Some of the projects and dependencies we will be working on cleaning are:
 
-* SpringWeb
-* Pandas
-* SpringWeb Security 
-* Jackson-Databind
+* springweb
+* pandas
+* springWeb security 
+* jackson-databind
```

Comment:
please remove jackson-databind, additionally please resolve the conflicts to be merged.